src/Engines/thinEngine.ts

```diff
@@ -171,6 +171,29 @@ export class ThinEngine {
         Effect.ShadersRepository = value;
     }
 
+     /**
+     * Gets or sets the textures that the engine should not attempt to load as compressed
+     */
+    public static ExcludedCompressedTextures: string[] = [];
+
+    /**
+     * Filters the compressed texture formats to only include
+     * files that are not included in the skippable list
+     *
+     * @param url the current extension
+     * @param textureFormatInUse the current compressed texture format
+     * @returns "format" string
+     */
+    public static ExcludedCompressedTextureFormats(url: Nullable<string>, textureFormatInUse: Nullable<string>): Nullable<string> {
+        const skipCompression = (): boolean => {
+            return ThinEngine.ExcludedCompressedTextures.some((entry) => {
+                const strRegExPattern: string = '\\b' + entry + '\\b';
+                return (url && url) === entry || (url && url.match(new RegExp(strRegExPattern, 'g')));

```

Comment:
(url && url) === entry could we just not use url === entry ?

Reply:
We don't want this method to return false if there are null values, so we only want to do this test if both `url` and the exclusion string are populated.

Reply:
but what does it change for (url && url) === entry I am still confuse I am sorry :-) 

isnt (null && null) equal to null and any none null value equal to themselves ?

Reply:
Oh shoot I think the paren got moved. I'll update, sorry.